config/tags.config

```diff
@@ -28,6 +28,11 @@ ubuntu-jdk-releases-slim-tags: slim {{ JDK_RELEASES_VER }}-slim {{ ARCH }}-ubunt
 ubuntu-jdk-nightly-full-tags: nightly {{ JDK_NIGHTLY_VER }}-nightly {{ ARCH }}-ubuntu-{{ JDK_NIGHTLY_VER }}-nightly
 ubuntu-jdk-nightly-slim-tags: nightly-slim {{ JDK_NIGHTLY_VER }}-nightly-slim {{ ARCH }}-ubuntu-{{ JDK_NIGHTLY_VER }}-nightly-slim
 
+debian-jdk-releases-full-tags: latest {{ JDK_RELEASES_VER }} {{ ARCH }}-debian-{{ JDK_RELEASES_VER }}
+debian-jdk-releases-slim-tags: slim {{ JDK_RELEASES_VER }}-slim {{ ARCH }}-debian-{{ JDK_RELEASES_VER }}-slim
+debian-jdk-nightly-full-tags: nightly {{ JDK_NIGHTLY_VER }}-nightly {{ ARCH }}-debian-{{ JDK_NIGHTLY_VER }}-nightly
+debian-jdk-nightly-slim-tags: nightly-slim {{ JDK_NIGHTLY_VER }}-nightly-slim {{ ARCH }}-debian-{{ JDK_NIGHTLY_VER }}-nightly-slim
+
```

Comment:
There can only be one `latest`, `slim`, `nightly` and `nightly-slim`. It is set for ubuntu images right now. The debian ones need to be similar to the alpine ones with the `debian` preprended to it.

Reply:
should be ok now, happy if you overview that one once again though - tricky strings :)